animeal/src/Flows/Main/Modules/Home/FeedingPointDetails/Model/FeedingPointDetailsModel.swift

```diff
@@ -42,7 +42,6 @@ final class FeedingPointDetailsModel: FeedingPointDetailsModelProtocol, FeedingP
         self.feedingPointId = pointId
         self.mapper = mapper
         self.context = context
-        subscribeForFeedingPointChangeEvents()
```

Comment:
Removing update might negatively affect various edge cases (e.g. when feeding point updated while user is on details screen).

I would suggest looking into another solution based on comparing data change for image and preventing its reload if it is the same one.